src/components/BooksList.js

```diff
@@ -1,35 +1,40 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
-import { removeBook } from '../redux/books/books';
+import { deleteBook, getBooksData } from '../redux/books/bookthunk';
 import Book from './Book';
 
```

Comment:
The app does not follow style guidelines. Kindly follow the design and implement it according to guidelines provided.

Yours:
![yours](https://user-images.githubusercontent.com/35504149/174408501-cb7d1c24-b64e-42fa-a7c2-b06322cb0e52.png)

Design:

![app](https://user-images.githubusercontent.com/35504149/174408552-05cd3f97-dc7e-4f95-8282-f2417b6b5089.png)

Reply:
Hello @SajjadAhmad14 ! I mistakenly sent this PR for review in the wrong field (day) and immediately tried to undo the submission, but it told me it could not be done since someone had taken it for review. (All this happened seconds apart).
By my PR title and description, you can tell that this PR is about fetching data and not styling.
I would have wished not to lose one submission in vain, or to have been contacted in slack to clarify this. Anyway, I agree with you that the styling is not the same. And neither is the Title or description of the PR.